docs/preview/features/logging.md

```diff
@@ -111,6 +111,14 @@ public class Startup
             // All omitted HTTP request header names that should always be excluded from the request tracking logging emits.
             // (default: `[ "Authentication", "X-Api-Key", "X-ARR-ClientCert" ]`)
             options.OmittedRequestHeaderNames.Add("X-My-Secret-Header");
+
+            // Size of the request body buffer (in bytes) which should be tracked. Request bodies greater than this buffer will only be partly present in the telemetry.
+            // (default: 1MB)
+            options.RequestBodyBufferSize = 10 * 1024 * 1024; // 10MB
```

Comment:
I presume this is optional?

Reply:
Yes, it's default on 1MB, which means only a MB on data is going to be send to the tracking system (which is already a lot for a single request I think).

Reply:
Question is, why bother?